pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -97,18 +109,38 @@ func New(
 		pkgmetrics.Record(reporterCtx, panicM.M(0))
 	}
 
+	revisionObj, err := revisionLister.Revisions(namespace).Get(revision)
+	if err != nil {
+		return nil, errors.New("unable to get revision" + revision)
+	}
+	var initScale *int
+	if revInitScale, ok := revisionObj.Annotations[autoscaling.InitialScaleAnnotationKey]; ok {
```

Comment:
We should not be looking for this on the Revision.  Where do we pull `{min,max}Scale`?

Reply:
min/maxScale are pulled in `pkg/reconciler/autoscaling/kpa/scaler.go`. However I ran into a problem when I only implement this there. We want to scale down from initial scale once we reach stable window. However, `scaler.Scale(...)` doesn't get triggered at that point since there's no change in PA or metrics, so we cannot scale down from initial scale.